src/Uno.UI/UI/Xaml/UIElement.wasm.cs

```diff
@@ -729,5 +733,25 @@ private static IEnumerable<string> GetClassesForType(Type type)
 				}
 			}
 		}
+
+		private InputCursor _protectedCursor = InputSystemCursor.Create(InputSystemCursorShape.Arrow);
+
+		protected InputCursor ProtectedCursor
+		{
+			get => _protectedCursor;
+			set
+			{
+				_protectedCursor = value;
+				Internal_SetProtectedCursor();
```

Comment:
Do we want to call this only if `_protectedCursor != value`?

Reply:
```suggestion
				SetProtectedCursorNative();
```